### ISSUE TITLE
Allow for `orderby` in Gallery block

### ISSUE THREAD

**Issue by cjyabraham** [WordPress/gutenberg#15370]
The gallery shortcode has an `orderby` parameter.  Is it possible to move that functionality into the Gallery Block workflow?  Perhaps the best place to put it is during the "Edit Gallery" phase where you can manually order the images.  At that point you could have a drop-down above which allows for sorting by a field and then allow manual override after.

**Comment by draganescu** [comment]
Thinking about this, while it looks priority low, I would say photo intensive sites could use default sorting by title or upload date or other image meta. However it does look like a plugin wouldl solve this a lot better than baking features into core.

Is this a feature the core gallery block should support (aside from the fact that the shorcode does)?

**Comment by af1** [comment]
> Thinking about this, while it looks priority low, I would say photo intensive sites could use default sorting by title or upload date or other image meta. However it does look like a plugin wouldl solve this a lot better than baking features into core.
> 
> Is this a feature the core gallery block should support (aside from the fact that the shorcode does)?

Yes. I want to sort the images and there's no way to do so. If I could sort, I could use the Gallery Block, sort, then convert them to single image blocks and be done. I could avoid: me having to use the classic editor, edit the HTML, copy out to Sublime Text, sort the lines, and paste back in.
